### PR TITLE
Reader: add tracks stats for Following Intro banner

### LINKED ISSUE #13756: Reader: add stats for Following Intro banner
We need to fire stats for:
- banner impression (like we did with combined cards) 
- close banner

screen shot 2017-05-08 at 15 03 16

### PR DESCRIPTION
**Pull request by bluefuton**
Adds two new Tracks stats for the Following Intro banner - one on render, and one on close/dismiss.

Fixes #13756.

bad93790-33ff-11e7-932e-440920f4860c

### To test

Create a brand new user. Make sure the render event is fired when the following intro banner loads (and only once per page load). Make sure the dismiss event fires when the close button is clicked.

screen shot 2017-05-09 at 16 48 36

### CODE REVIEW

File: client/reader/following/intro.jsx [hunk 1]
```diff
@@ -13,44 +13,65 @@ import { bindActionCreators } from 'redux';
...
+				<div className="following__intro-header">
+					<div className="following__intro-copy">
+							{ translate(
+								'{{strong}}Welcome!{{/strong}} Reader is a custom magazine. ' +
+								'{{link}}Follow your favorite sites{{/link}} and their latest ' +
+								'posts will appear here. {{span}}Read, like, and comment in a ' +
+								'distraction-free environment.{{/span}}',
+								{
+									components: {
+										link: <a href="/following/edit" />,
```
Comment (blowery) [inline client/reader/following/intro.jsx]: maybe this should respect the feature flag with https://github.com/Automattic/wp-calypso/pull/13779 landing?

File: client/reader/following/intro.jsx [hunk 2]
```diff
@@ -13,44 +13,65 @@ import { bindActionCreators } from 'redux';
...
+							) }
+					</div>
 
-				<div className="following__intro-close"
-					onClick={ props.dismiss }
-					title={ props.translate( 'Close' ) }
-					aria-label={ props.translate( 'Close' ) }>
-						<Gridicon icon="cross-circle" className="following__intro-close-icon" title={ props.translate( 'Close' ) } />
-						<span className="following__intro-close-icon-bg" />
+					<div className="following__intro-close"
```
Comment (blowery) [inline client/reader/following/intro.jsx]: should this be a `<button>` instead of a div for a11y? I'm asking mostly out of ignorance. :)

**Comment by blowery** [comment]
Left a couple comments unrelated to the tracks recording. Tracks looks good.

File: client/reader/following/intro.jsx (hunk 2 above)
Comment (bluefuton) [inline client/reader/following/intro.jsx]: It very likely should! Nicer semantically and for a11y.

Converting to `<button>` unfortunately scuppers our existing styles, so I've added the attribute `role="button"`, which covers the a11y requirement.

File: client/reader/following/intro.jsx (hunk 1 above)
Comment (bluefuton) [inline client/reader/following/intro.jsx]: Good plan. It now respects the feature flag and changes the link href accordingly.

**Pull request merged**